### DISCUSSION TITLE
[Feature Request] Can I change the style of the TAB

### DISCUSSION THREAD

**Discussion by q812143836** [microsoft/monaco-editor#3806] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

![TabStyle](https://user-images.githubusercontent.com/54973061/190719901-5878e73f-f4b4-4375-821e-feb3759ea678.jpg)
Let's say I want to change it to: "➽" "⟶" “|” or whatever symbol I like

**Comment by tamayika** [comment]
I digged vscode codebase and found svg is written manually, so it is difficult to change it.
https://github.com/microsoft/vscode/blob/main/src/vs/editor/browser/viewParts/whitespace/whitespace.ts#L129-L267
